src/App.scss

```diff
@@ -148,6 +162,20 @@ body {
       cursor: pointer;
       width: 100%; //height: 87px;
    }
+
+   #hamburger {
+      grid-row: 3;
+      grid-column: 3;
+      cursor: pointer;
+      width: 10%; 
+   }
+
+   #helpimg1 {
+      grid-row: 3;
+      grid-column: 3;
+      cursor: pointer;
+      width: 80%; 
```

Comment:
this image is cut off in the y axis on my screen

Reply:
I've changed the width to 60%. I know we havnt changed it to conditional rendering but could you approve the change for now, and we can look at it sprint 3?